src/default_panels/StyleTracesPanel.js

```diff
@@ -241,6 +241,22 @@ const StyleTracesPanel = (props, {localize: _}) => (
       <SymbolSelector label={_('Symbol')} attr="marker.symbol" />
       <Numeric label={_('Border Width')} attr="marker.line.width" />
       <ColorPicker label={_('Border Color')} attr="marker.line.color" />
+      <Radio
```

Comment:
can you plz remove these from `line` given that we don't allow line colorscales atm :)